include/nifty/graph/rag/get_lifted_edges_from_rag_and_offsets.hxx

```diff
@@ -0,0 +1,98 @@
+#pragma once
+
+#include <random>
+#include <thread>
+#include "xtensor/xexpression.hpp"
+
+#include "xtensor/xview.hpp"
+#include "xtensor/xmath.hpp"
+#include "nifty/graph/undirected_list_graph.hxx"
+
+#include "nifty/array/arithmetic_array.hxx"
+#include "nifty/tools/for_each_coordinate.hxx"
+#include <cstdlib>
+
+#include <pybind11/pybind11.h>
+namespace py = pybind11;
```

Comment:
Why do you have the pybind11 namespace here in `include`? Ideally it should be only used in `src/python`.

Reply:
I added it because I used `py::gil_scoped_release` (see comment below). If we find a better solution for that, we can remove this include.

Reply:
This should be easily avoided, see comment below.